pkg/cloudprovider/cloudprovider.go

```diff
@@ -145,31 +155,28 @@ func (c *CloudProvider) LivenessProbe(req *http.Request) error {
 func (c *CloudProvider) GetInstanceTypes(ctx context.Context, provisioner *v1alpha5.Provisioner) ([]*cloudprovider.InstanceType, error) {
 	var nodeTemplate *v1alpha1.AWSNodeTemplate
 	var err error
```

Comment:
Should be able to bring not define these here and just use the shorthand assignment below now.

Reply:
Good eye